.github/workflows/pytest.yml

```diff
@@ -60,6 +60,17 @@ jobs:
         sh install.sh -y
         cargo install project_ares
 
+    - name: Install Shuriken-Analyzer
+      run: |
+        sudo apt-get install -y cmake git default-jdk iputils-ping
+        git clone https://github.com/Shuriken-Group/Shuriken-Analyzer.git /opt/Shuriken-Analyzer
+        cd /opt/Shuriken-Analyzer
+        cmake -S . -B build/ -DCMAKE_BUILD_TYPE=Release -DGITHUB_ACTION=ON
+        cmake --build build/ -j
+        sudo cmake --install build/
+        pip install /opt/Shuriken-Analyzer/shuriken/bindings/Python/
+        cd -
```

Comment:
Shuriken has fixed the installation issue with Pip (See [here](https://github.com/Shuriken-Group/Shuriken-Analyzer/issues/99)). We can simply install Shuriken via Pip now.